icevision/models/interpretation.py

```diff
@@ -223,7 +224,7 @@ def plot_top_losses(
         samples = add_annotations(samples)
 
         dl = self.infer_dl(dataset, batch_size=batch_size)
-        preds = self.predict_from_dl(model=model, infer_dl=dl)
+        preds = self.predict_from_dl(model=model, infer_dl=dl, keep_images=True)
```

Comment:
Why do we need `keep_images` here?
We rebuild `Prediction` objects from `sorted_samples` [here](https://github.com/airctic/icevision/pull/934/files#diff-f3f81ce939d27f5cf5ed326cf50fc5cf26d47b9d1cf92ac04a22d45c802b4807R256) which should have `img`s set properly already.

Reply:
I can't double check right now but I _think_ this is related to the fact that without `keep_images` set to `True`, GT masks would not be available for subsequent operations. Happy to revisit this one if you think it is needed.

Reply:
You are right that the masks would not be available in GT, given the records get unloaded.
I think this is not a problem specific to the `plot_top_losses` functionality as we get GT records from [here](https://github.com/airctic/icevision/pull/934/files/6de58ba0fcd3f1af7d9c6981522b68615fe92d8c#diff-f3f81ce939d27f5cf5ed326cf50fc5cf26d47b9d1cf92ac04a22d45c802b4807R223). We don't take them from the `preds` object. As you can see [here](https://github.com/airctic/icevision/pull/934/files/6de58ba0fcd3f1af7d9c6981522b68615fe92d8c#diff-f3f81ce939d27f5cf5ed326cf50fc5cf26d47b9d1cf92ac04a22d45c802b4807R228) from `preds` we get only the predictions. Long story short the GT records are the same we had ad inputs in the dataset, therefore they are not unloaded. They have everything set properly.

Reply:
I remember adding this due to tests related to `plot_top_losses` failing - maybe it isn't actually needed?

Reply:
I'd be surprised if it was needed. Can you check removing it and see what happens?

Reply:
I removed it, and tests still work. Doesn't seem to be break anything, so will keep it like that for now!

Reply:
Thanks for checking man!